pages/web_cloud/email_and_collaborative_solutions/email_pro/first_config/guide.en-gb.md

```diff
@@ -27,67 +27,85 @@ To do this, log in, then click on `Email Pro`{.action} and select the appropriat
 
 Once you have selected an option, click on the `Next`{.action} button.
 
-![emailpro](images/first_config_email_pro_add_domain.png){.thumbnail}
+![emailpro](images/emailpro-01.png){.thumbnail}
 
-The window will then appear, showing information on configuring a mode.
+The window will then show information on configuring modes.
 
-- **If you have entered a non-OVHcloud domain name**: Non-authoritative mode will be configured by default.
-- **If you have selected an OVHcloud domain name in the list**: you must choose between two modes.
+![emailpro](images/emailpro-02.png){.thumbnail}
 
-|Mode|Description|
-|---|---|
-|Authoritative|Choose this if you only use the Email Pro solution with your domain name. In authoritative mode, you cannot use another email solution with your Email Pro service.|
-|Non-authoritative|Choose this if you use your Email Pro solution domain name with another email solution.| 
+- **If you have entered a non-OVHcloud** domain name, non-authoritative mode will be configured by default.
 
-> [!primary]
+- **If you have selected a domain name managed by OVHcloud** from the list, you will need to choose between two modes.
+    - **Authoritative**: Choose this if your Email Pro solution is the only email solution you use with your domain name. It does not allow the use of another email solution with your service.
+    - **Non-authoritative**: Choose this if you use the Email Pro solution and **another email solution** that uses the same domain name.
+
+> **Understanding Authoritative and Non-Authoritative Modes**
 >
-> The mode choice is not definitive. It can be modified via the OVHcloud Control Panel later on.
+> - When an email is sent to your Email Pro (*Inbound mail server Email Pro*) platform in **authoritative** mode, this means that all of your domain name’s email addresses are only hosted on this platform. <br> <br> For example, if you send an email to the address *mary.johnson@mydomain.ovh*, the Email Pro server *Inbound mail server Email Pro* will send a failure message to the sender, because that address does not exist on the Email Pro server "*Inbound mail server Email Pro*".
+> - When an email is sent to your Email Pro (*Inbound mail server Email Pro*) email platform in **non-authoritative** mode, this means that the email addresses for your domain name are divided between your main email platform (*Inbound mail server Email Pro*) and another email service (*Inbound mail server MX Plan*). <br> <br> For example, if you send an email to the address *mary.johnson@mydomain.ovh*, the Email Pro "*Inbound mail server Email Pro*" will send the email to the MX Plan2 server *Inbound mail server MX Plan233* so that it can deliver it.
```

Comment:
where do these numbers come from ?
"For example, if you send an email to the address *mary.johnson@mydomain.ovh*, the Email Pro "*Inbound mail server Email Pro*" will send the email to the MX Plan2 server *Inbound mail server MX Plan233* so that it can deliver it."